org.libsdl.Maelstrom.yml

```diff
@@ -0,0 +1,43 @@
+app-id: org.libsdl.Maelstrom
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: maelstrom
+rename-icon: maelstrom
+rename-desktop-file: maelstrom.desktop
+rename-appdata-file: maelstrom.appdata.xml
+
+finish-args:
+  - --device=dri
+  - --share=ipc
+  - --share=network
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --socket=pulseaudio
+
+modules:
+  - name: maelstrom
+    buildsystem: autotools
+    post-install:
+      - mkdir -p ${FLATPAK_DEST}/bin
+      - ln -sf ${FLATPAK_DEST}/games/Maelstrom/Maelstrom ${FLATPAK_DEST}/bin/maelstrom
+      - install -Dm644 maelstrom.png ${FLATPAK_DEST}/share/icons/hicolor/512x512/apps/maelstrom.png
+      - install -Dm644 maelstrom.desktop ${FLATPAK_DEST}/share/applications/maelstrom.desktop
+      - install -Dm644 maelstrom.appdata.xml ${FLATPAK_DEST}/share/appdata/maelstrom.appdata.xml
```

Comment:
Since you install everything, you should directly have the file with the right content and installed with the right name.
`rename` is really when everything if from upstream, or when there are changes to do to the files supplied by upstream.